### ISSUE TITLE
Recipe Step-By-Step Instructions

### ISSUE THREAD

**Issue by fcpranav** [ECE444-2020Fall/project1-plateup-group5-pyromaniacs#52]
The user should be able to click the "Let's go!" button on the recipe preview screen to see step-by-step instructions on how to make a specific recipe. If that screen is not yet implemented, develop this screen independently and coordinate with the other developer as needed.

AC:
- Design matches Figma
- User should be able to see step-by-step instructions on how to create a particular recipe
- This functionality works with both mock data and real data

Note:
- Feel free to create a follow-up task if that is what is needed to work on this ticket in parallel with #51 

Figma: https://www.figma.com/file/oxcmkYmcZGhSXuUaSBKUjo/Revision-1?node-id=115%3A404
